include/DoViProcessor.h

```diff
@@ -6,7 +6,10 @@
 
 #include "rpu_parser.h"
 #include <vector>
+#ifdef _WIN32
```

Comment:
When including windows.h it's better to exclude some APIs to avoid conflicts with standard library.

These should be added:
```
#ifdef _WIN32
#ifndef NOMINMAX
#define NOMINMAX
#endif
#ifndef WIN32_LEAN_AND_MEAN
#define WIN32_LEAN_AND_MEAN
#endif
```